concordium-sdk/src/main/java/com/concordium/sdk/transactions/Transaction.java

```diff
@@ -1,63 +1,52 @@
 package com.concordium.sdk.transactions;
 
-import com.concordium.sdk.exceptions.TransactionCreationException;
+import com.concordium.sdk.crypto.SHA256;
 
-import java.util.Objects;
+import static com.google.common.primitives.Bytes.concat;
 
 public interface Transaction {
+    int DEFAULT_NETWORK_ID = 100;
+    // todo: in practice this is true, but in the future this could different as the version is variable length encoded.
+    int VERSION_SIZE = 1;
+    int VERSION = 0;
+
     /**
      * Returns serialized {@link Transaction}
      * This is the raw bytes that is sent to the node.
+     * It is a concatenation of the Transaction Version {@link Transaction#VERSION} + {@link Transaction#getTransactionRequestPayloadBytes()}
      *
      * @return The serialized {@link Transaction}
      */
-    byte[] getBytes();
+    default byte[] getBytes() {
+        return concat(new byte[]{(byte) VERSION}, getTransactionRequestPayloadBytes());
+    }
+
+    /**
+     * Transaction serialized as bytes.
+     * How a transaction is serialized to bytes depends on type of the transaction. See {@link BlockItemType}.
+     * <p>These bytes are not sent directly to the node. For that {@link Transaction#getBytes()} is used.</p>
+     *
+     * @return Serialized bytes of this Transaction.
+     */
+    byte[] getTransactionRequestPayloadBytes();
```

Comment:
And then this one i.e. `getTransactionRequestPayloadBytes` could just be `getBytes`.

Reply:
done. Thats exactly what we did initially. Can be checked from previous commits. 
However since `Transaction.getBytes` is a public method we didnt want to change the functionality of that particular method.